packages/tables/resources/views/index.blade.php

```diff
@@ -42,6 +42,7 @@
     $isReorderable = $isReorderable();
     $isReordering = $isReordering();
     $areGroupingSettingsVisible = (! $isReordering) && count($groups) && (! $areGroupingSettingsHidden());
+    $isGroupingDirectionHidden = (! $isGroupingDirectionHidden());
```

Comment:
This doesn't make logical sense with the !, the value is true if its actually false

Reply:
You'll need to swap around the `@if` statements in the groups.blade.php view to fix the change as well

Reply:
Or, a better alternative could be renaming to `$isGroupingDirectionVisible` and passing the groups.blade.php attribute as just `:direction="true / false"` which I think is still clear?

Reply:
Updated as suggested, please check

Reply:
Thanks